docker-for-mac/apple-m1.md

```diff
@@ -1,16 +1,64 @@
 ---
-description: Apple M1 Tech Preview
+description: Apple M1
 keywords: Docker Desktop, M1, Silicon, Apple, tech preview, dev preview
-title: Apple M1 Tech Preview
+title: Apple M1
 toc_min: 2
 toc_max: 3
 ---
 
-Welcome to the tech preview of Docker Desktop for Apple M1. This tech preview is aimed at early adopters of Apple M1 machines, who would like to try an experimental build of Docker Desktop.
+Welcome to Docker Desktop for Apple Silicon.
 
 > **Note**
 >
-> Docker Desktop on Apple M1 chip is still under development. We recommend that you do not use tech preview builds in production environments.
+> Docker Desktop for Apple Silicon is still under development. Please try these builds and report issues on [docker/for-mac](https://github.com/docker/for-mac).
+
+## Docker Desktop RC 1
+
+2021-03-18
+
+Click the following link to diwnload the Apple Silicon RC 1 build:
+
+> [Download](https://desktop.docker.com/mac/stable/arm64/62015/Docker.dmg)
+
+## Known issues
+
+The following issues are known and are not expected to be resolved in the final GA build for M1.
```

Comment:
The two lists of bullets are confusing. I don't understand which items belong in which list. It's not ones we need to fix vs ones Apple needs to fix, nor is it ones that might be fixed later vs ones that will never be fixed.

I suggest just having a single list with rubric such as:

> The following known issues are not expected to be resolved in the final GA build for M1. We are working on them in collaboration with our partners.

And then note within each bullet whether we expect to fix it later (as you already have done).